packages/apps/web-viewer-test/src/components/home/ViewerHome.tsx

```diff
@@ -125,16 +154,19 @@ const ViewerHome: React.FC = () => {
       groupPriority: 20,
       itemPriority: 100,
       label: "BackstageItems 1",
-    }
+    },
   ];
 
   return (
     <div style={{ height: "100vh" }}>
       <Viewer
         authClient={authClient}
-        iTwinId={iTwinId ?? ""}
-        iModelId={iModelId ?? ""}
-        changeSetId={changesetId}
+        iTwinId={iTwinId}
+        iModelId={iModelId}
```

Comment:
Unlike iTwinId, iModelId hasn't been updated to default to empty string. It could be that the old code was broken, and it could be that other changes you made make it so undefined is the desired value here, but it could also be that this change is a mistake.

Reply:
I've set all to themselves or an empty string now, thanks.

Reply:
Is the trailing `&` at the end of the new url OK, or should it be removed after computing the url?

Reply:
Right, yeah I think it's okay.